pkg/util/utils.go

```diff
@@ -692,3 +693,16 @@ func CoresToPeriodAndQuota(cores float64) (uint64, int64) {
 func PeriodAndQuotaToCores(period uint64, quota int64) float64 {
 	return float64(quota) / float64(period)
 }
+
+// IdtoolsToRuntimeSpec converts idtools ID mapping to the one of the runtime spec.
+func IdtoolsToRuntimeSpec(idMaps []idtools.IDMap) (convertedIDMap []specs.LinuxIDMapping) {
```

Comment:
Suggest renaming to `IDtoolsToRuntimeSpec` (Capitalize the first D) - I think our linter checks for this one now. Might be it's not linting this directory?

Reply:
Thanks and done